packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -29,8 +32,14 @@ describe('Validation at REST level', () => {
     @property({required: true})
     name: string;
 
-    @property({required: false})
-    description?: string;
+    // We have to add `type: 'string'` as `string | null` removes TypeScript
+    // design:type reflection
+    @property({required: false, jsonSchema: {nullable: true, type: 'string'}})
+    description?: string | null;
+
+    // The other option is to define `string` instead `string | null`
```

Comment:
Can `null` be assigned to `string` typed variables/properties? Will the following compile?

```ts
product.note = null;
```

If it does not compile, then I think your proposal is wrong.

Reply:
I removed `note`.